### PR TITLE
[CAP-76] - Implement forgot password

### PR DESCRIPTION
**Pull request by banhsbao**
- Implement forgot password

### CODE REVIEW

File: jobfair-frontend/src/router/AppRouter.js [hunk 1]
```diff
@@ -1,155 +1,155 @@
...
+          <JobFairListPage />
+        </Route>
+        <Route path="/choose-booth/:jobFairId" exact>
+          <ChooseBoothPage />
+        </Route>
+        <Route path="/proceed-success" exact>
+          <ResultSuccessPage />
+        </Route>
+        <AttendantRouter key="/applied-job" component={<AppliedJobPage />} path="/applied-job" exact />
+        <CompanyManagerRouter
```
Comment (CaoSon1909) [inline jobfair-frontend/src/router/AppRouter.js:103]: we will discuss about this later

**Review by CaoSon1909 — APPROVED** [review]
please change

**Review by CaoSon1909 — DISMISSED** [review]
please change Mr.BanhsBao 👍🏾

**Review by ZERO-94 — APPROVED** [review]

**Comment by banhsbao** [comment]
@ZERO-94  take a look

**Review by ZERO-94 — APPROVED** [review]

**Pull request merged**